node/src/cli.rs

```diff
@@ -129,6 +137,11 @@ impl Arguments {
             ..run_cmd
         }
     }
+
+    pub fn block_author(&self) -> AccountId {
+        self.block_author
+            .unwrap_or_else(|| AccountId::from_raw([0; 32]))
+    }
```

Comment:
suggestion:

```suggestion
        let account_zero = AccountId::from_raw([0; 32]);
        self.block_author.unwrap_or(account_zero)
    }
```

I'd like to name the account zero and maybe even have it aside and explained, if you see the value. I happen to know hat it is from a conversation with Alexis, wouldn't have a clue otherwise.

And no need to `unwrap_or_else(|| ...)`, `unwrap_or` is sufficient.